edx_proctoring/tests/test_api.py

```diff
@@ -1878,6 +1878,39 @@ def test_update_exam_attempt(self):
         self.assertEqual(attempt['last_poll_ipaddr'], '1.1.1.1')
         self.assertEqual(attempt['time_remaining_seconds'], 600)
 
+    def test_update_exam_attempt_ready_to_resume(self):
+        exam_attempt = self._create_started_exam_attempt()
+
+        # Assert that state transition from a non-error state to a ready_to_resume state raises
+        # an exception.
+        with self.assertRaises(ProctoredExamIllegalStatusTransition):
+            update_attempt_status(
+                exam_attempt.proctored_exam_id,
+                self.user.id,
+                ProctoredExamStudentAttemptStatus.ready_to_resume
+            )
+
+        attempt = get_exam_attempt_by_id(exam_attempt.id)
```

Comment:
#NIT Can we have the test blob below into a new test case? This way, we can make the tests easier to read.